web/template/home.gohtml

```diff
@@ -491,8 +491,17 @@
 
                                                                         </a>
                                                                     </template>
+                                                                    <template x-if="vod.HasName()">
                                                                     <span class="date text-sm"
                                                                           x-text="vod.FriendlyDateStart()"></span>
+                                                                    </template>
+
+                                                                    <template x-if="!vod.HasName()">
+                                                                        <a :class="isListView() ? 'title overflow-hidden' : ''" :href="course.WatchURL(vod.ID)">
+                                                                        <span class="date" :class="isListView() ? '' : 'text-sm'"   
+                                                                              x-text="vod.FriendlyDateStart()"></span>
+                                                                        </a>
+                                                                    </template>
                                                                 </div>
                                                                 <button type="button" @click="vod.Dropdown.toggle()"
```

Comment:
This button is currently too far away from the title... This makes it hard for me to notice that there's a button. Do you think it would be better to place it more to the left?
![image](https://github.com/TUM-Dev/gocast/assets/78721605/2a3fdeaa-6e0c-432a-a722-492e86bf1c3f)

Reply:
Where would you put it?

Reply:
That's true... And I don't think this button is very frequently used (those who click on them a lot will probably find them anyway). I'll approve the change and maybe you can deploy this fix along with (after) the test tonight.